### ISSUE TITLE
Change the UI of the extension. 

### ISSUE THREAD

**Issue by ShreyasheeS** [ShreyasheeS/Open-Multiple-Websites#1]
Minor changes required for the UI of the chrome extension, eg: the background colour, font, buttons etc.

**Comment by filoxo** [comment]
I'd like to take this on! Any thoughts on a specific design system? If not, maybe material design?

**Comment by rishiosaur** [comment]
@filoxo Yeah, I guess. It would go pretty well, seeing as google chrome uses it as well.

**Comment by ShreyasheeS** [comment]
@filoxo Yes! Material Design would be great!

**Comment by rishiosaur** [comment]
#3 @filoxo already made the MD version of this, but I think that we should add more themes, like a night version. Let me know what you think of this!

**Comment by justintime4tea** [comment]
Why not use an MD library for the style? This way we won't have to re-build the wheel and can make use of MD panels, animations, and other MD styling/elements. 

Theme's could be provided as CSS "overrides" which load after the MD CSS and only need to define/override color attributes.

With using an MD library it would be fairly trivial to implement issue #2 by using MD tabs at the top choosing between copy/paste and a linked list of history items.

**Comment by filoxo** [comment]
Reaching for a full style framework is like reaching for a sledgehammer to tack in a small nail. The requirements haven't warranted one at the time of this issue. Plus... The current UI is literally a heading, a textarea, and a button. Definitely would be overkill to have brought in a framework for that.

Implementing #2 will be trivial without a framework too, depending on what requirements have been decided on.

**Comment by justintime4tea** [comment]
MDL comes in at about 200KB, hardly a sludge hammer and it helps you actually conform to Material instead of imitating Material design.

Making even just a simple MD tabbed bar or other fancy MD elements is a lot of work that's already been done. Just adding my two sense/opinion. We're all entitled to disagree or agree. 

If all that's needed is a couple UI elements, you don't want/need cool ripple effects and other MD animations and there's no plan to make a more extensive UI then sure, rebuilding the wheel isn't much work at all.

Going to go ahead and apply a lightweight MD library anyways and if the author doesn't want to incorporate it that's fine. That's the beauty of open source. It never has to be done only one way :)
